public/js/modal.js

```diff
@@ -0,0 +1,3 @@
+window.onload = () => {
+
+};
```

Comment:
why is it empty?
see https://github.com/BaSD-2021/Esteban-MindSet-server/pull/44#discussion_r753704183

Reply:
fixed!